src/test/java/gov/loc/repository/bagit/verify/BagVerifierTest.java

```diff
@@ -118,6 +120,10 @@ public void testVersion0_97IsValid() throws Exception{
   @Test
   public void testVersion0_98IsValid() throws Exception{
     rootDir = Paths.get(getClass().getClassLoader().getResource("bags/v0_98/bag").toURI());
+    if (System.getProperty("os.name").contains("Windows")){
+    	Path bagitDir = rootDir.resolve(".bagit");
```

Comment:
a tag manifest isn't meant to be listed in any manifest. This is a bug - as @acdha pointed out once we have a windows CI we will be able to better detect and fix them